src/components/common/CardImage/CardImage.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
+import { useMediaQuery } from 'react-responsive';
 
-const CardImage = ({ url, alt, height, width }) => {
+const CardImage = ({ url, alt, height, width, smheight }) => {
+  const sm = useMediaQuery({ maxWidth: '600px' });
```

Comment:
Don't you think making responsive layout here instead CSS would be much easier you import react response make const and use to check the width and based on that alter height of a div 
Css could have done it much faster

Reply:
But liked the different approach

Reply:
Since, the Image height depends on height prop, that's why this way. Adding fixed height through CSS would remove the flexibility of passing any height value based on different requirements.